fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockTags.java

```diff
@@ -0,0 +1,112 @@
+	 */
+	public static final TagKey<Block> BUDS = register("buds");
+	/**
+	 * For blocks that are similar to amethyst where they have clusters forming from budding blocks.
+	 */
+	public static final TagKey<Block> CLUSTERS = register("clusters");
+
+	public static final TagKey<Block> VILLAGER_JOB_SITES = register("villager_job_sites");
+
+	// Sandstone
+	public static final TagKey<Block> BLOCKS_SANDSTONE = register("sandstone/blocks");
+	public static final TagKey<Block> SLABS_SANDSTONE = register("sandstone/slabs");
+	public static final TagKey<Block> STAIRS_SANDSTONE = register("sandstone/stairs");
+	public static final TagKey<Block> RED_BLOCKS_SANDSTONE = register("sandstone/red_blocks");
+	public static final TagKey<Block> RED_SLABS_SANDSTONE = register("sandstone/red_slabs");
+	public static final TagKey<Block> RED_STAIRS_SANDSTONE = register("sandstone/red_stairs");
+	public static final TagKey<Block> UNCOLORED_BLOCKS_SANDSTONE = register("sandstone/uncolored_blocks");
+	public static final TagKey<Block> UNCOLORED_SLABS_SANDSTONE = register("sandstone/uncolored_slabs");
+	public static final TagKey<Block> UNCOLORED_STAIRS_SANDSTONE = register("sandstone/uncolored_stairs");
+
+	// Blocks created with dyes
+	public static final TagKey<Block> DYED = register("dyed");
+	public static final TagKey<Block> BLACK_DYED = register("dyed/black");
+	public static final TagKey<Block> BLUE_DYED = register("dyed/blue");
+	public static final TagKey<Block> BROWN_DYED = register("dyed/brown");
+	public static final TagKey<Block> CYAN_DYED = register("dyed/cyan");
+	public static final TagKey<Block> GRAY_DYED = register("dyed/gray");
+	public static final TagKey<Block> GREEN_DYED = register("dyed/green");
+	public static final TagKey<Block> LIGHT_BLUE_DYED = register("dyed/light_blue");
+	public static final TagKey<Block> LIGHT_GRAY_DYED = register("dyed/light_gray");
+	public static final TagKey<Block> LIME_DYED = register("dyed/lime");
+	public static final TagKey<Block> MAGENTA_DYED = register("dyed/magenta");
+	public static final TagKey<Block> ORANGE_DYED = register("dyed/orange");
+	public static final TagKey<Block> PINK_DYED = register("dyed/pink");
+	public static final TagKey<Block> PURPLE_DYED = register("dyed/purple");
+	public static final TagKey<Block> RED_DYED = register("dyed/red");
+	public static final TagKey<Block> WHITE_DYED = register("dyed/white");
+	public static final TagKey<Block> YELLOW_DYED = register("dyed/yellow");
+
+	/**
+	 * Blocks should be included in this tag if their movement can cause serious issues such as world corruption
+	 * upon being moved, such as chunk loaders or pipes, for mods that move blocks but do not respect
+	 * {@link AbstractBlock.AbstractBlockState#getPistonBehavior}.
+	 *
+	 * <p>See also the equivalent block entity type tag: {@link ConventionalBlockEntityTypeTags#RELOCATION_NOT_SUPPORTED}
+	 */
+	public static final TagKey<Block> RELOCATION_NOT_SUPPORTED = register("relocation_not_supported");
```

Comment:
[Tags are not intended to be a security/crash prevention feature.](https://bugs.mojang.com/browse/MC-260579) Please remove this.

Reply:
Disagree. It was added to the convention tag API for a reason.

Reply:
Indeed. There's already multiple mods already trying to mark their blocks are not able to be moved around in the world by tags. Makes sense to at least standardize it as people might still want to mark their block as not able to be relocated even in the absent of crashes or corruption. 
![image](https://github.com/FabricMC/fabric/assets/40846040/0f8f9979-5b7d-41ac-9678-feb8c80cfcf5)
![image](https://github.com/FabricMC/fabric/assets/40846040/2904834b-891f-4382-a1e4-7fd5c07095e4)

Reply:
I feel like if a mod is moving a block that shouldn't be moved it's on that mod to be careful about it, using tags for this isn't great because they can just be removed via datapacks...

Reply:
On the flip side, using tags for this also allows modpack makers to add more blocks to it if they find there's more problematic ones or want to restrict gameplay. And does give the option for pack makers to remove limits if the block was tagged because it was non-relocatable for balance reasons initially but pack makers wants to change that. It'll be on the pack maker to test and make sure it is all good but at least gives them the option.

Tag is also easier to handle multiloader than some sort of separate api that has to be hooked into for every loader and restricts ability for pack makers to add or remove blocks to see if it was blocked for corruption issues or add additional tech blocks that are having issues but wasn't blocked yet.